frontend/apps/marketing/src/config/utils.ts

```diff
@@ -0,0 +1,19 @@
+import {Brand} from '@/config/brand';
+import {getCanonicalHostname} from '@/config/host';
+import {Stage} from '@/config/stage';
+
+export function isExternalLink(href: string, brand: Brand, stage: Stage) {
```

Comment:
I'd prefer for this not to be in the config directory as it's not really a config.

Reply:
Sure!